src/app/app-routing.module.ts

```diff
@@ -3,11 +3,13 @@ import {RouterModule, Routes} from '@angular/router';
 import {AppComponent} from './app.component';
 import {UserLoginComponent} from './users/user-login/user-login.component';
 import {UserProfileComponent} from './users/user-profile/user-profile.component';
+import {PdfjsTestComponent} from './pdfjs-test/pdfjs-test.component';
```

Comment:
We shouldn't include test files and test code in the development branch. Please remove all test associated code and files.

Reply:
Fixed